src/utils/dom.ts

```diff
@@ -274,3 +274,7 @@ export function intersects(rect1: DOMRect, rect2: DOMRect): boolean {
     rect2.bottom < rect1.top
   );
 }
+
+export function toAriaBoolean(value: boolean): string {
```

Comment:
Could you add a comment about why we need this util? It won't be clear from our codebase, methinks.

Reply:
Yes